### PR TITLE
feat: change create table to accept Arrow table

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: node/src/query.ts [hunk 1]
```diff
@@ -129,7 +129,7 @@ export class Query<T = number[]> {
       const newObject: Record<string, unknown> = {}
       Object.keys(entry).forEach((key: string) => {
         if (entry[key] instanceof Vector) {
-          newObject[key] = (entry[key] as Vector).toArray()
+          newObject[key] = (entry[key] as Vector).toJSON()
```
Comment (wjones127) [inline node/src/query.ts:132]: Why this change?

**Review by wjones127 — COMMENTED** [review]
So many formatting changes 😢

File: node/src/query.ts (hunk 1 above)
Comment (eddyxu) [reply node/src/query.ts:132]: Arrow `toArray()` returns f16 array as uint16

Comment (wjones127) [reply node/src/query.ts:132]: That seems worth a comment.

Comment (eddyxu) [reply node/src/query.ts]: done
